src/jobs/build_push_image.yml

```diff
@@ -56,6 +56,10 @@ parameters:
     description: 'Steps that will run after build'
     type: steps
     default: [ ]
+  work_dir:
+    description: 'Working directory for build_push step'
+    type: string
+    default: .
```

Comment:
you should add this parameter to the [README](https://github.com/ricardo-ch/ricardo-orbs/blob/main/src/jobs/README.md?plain=1#L89-L104)

probably should also add an illuminating [example](https://github.com/ricardo-ch/ricardo-orbs/blob/main/src/jobs/README.md?plain=1#L108) that displays where/how you use this